docs/content/SRS/SRS.md

```diff
@@ -363,11 +503,43 @@ _The following Software Requirements Specification for `tinymorph` is using [Vol
 
 ### 24.1 User Documentation Requirements
 
-[Insert your content here.]
+UDT-D1. Create a user's guide that details operational procedures, user interfaces, and troubleshooting tips.
+
+Rationale: This guide will act as the primary resource for users to fully understand and effectively utilize all features of the application, enabling them to solve common problems independently. Keeping the guide updated will reflect ongoing changes and user feedback, enhancing user satisfaction and self sufficiency.
+
+UDT-D2. Develop an installation manual with detailed instructions for the software installation, configuration, and initial setup processes.
+
+Rationale: The manual ensures that all users regardless of their technical expertise can easily set up the application. This minimizes setup errors and facilitates smoother adoption of the software.
+
+UDT-D3. Produce an emergency procedure manual outlining critical steps for handling urgent software issues and unexpected operational scenarios.
```

Comment:
I think this is more prevalent once we shipped the zig binary, but probably fine to leave it here for now. 

let's move it a bit further down the list then?